src/sentry/event_manager.py

```diff
@@ -679,6 +679,18 @@ def _run_background_grouping(project, job):
         sentry_sdk.capture_exception()
 
 
+def _get_job_category(data):
+    event_type = data.get("type")
+    if event_type == "transaction":
+        # TODO: This logic should move into sentry-relay, but I'm not sure
+        # about the consequences of making `from_event_type` return
+        # `TRANSACTION_INDEXED` unconditionally.
```

Comment:
There is only one call to `from_event_type()` in sentry, 0 in getsentry, and 0 in snuba.

Reply:
👍  good catch. We use a similar function in Relay but rely on it to return `TRANSACTION` in this case. Let's keep it this way for now and I'll record a follow-up task to see how we can clean this up properly.